check_deps/check_python_dependencies.py

```diff
@@ -45,7 +45,7 @@
         imp.find_module(module)
         print("\t\tModule '%s' is installed" % module)
     except ImportError:
-        print("\t\tModule '%s' does not installed" % module)
+        print("\t\tModule '%s' is not installed" % module)
```

Comment:
There also some `doesn't installed` were which also should be fixed.

Reply:
Yeah, but "command is not installed" looks wrong anyway. So I need to do some rewording.

Reply:
I've changed the messages. Hope it's better this way.